src/components/MaStack/MaStack.vue

```diff
@@ -40,7 +45,7 @@ export default {
       type: [Array, String],
       required: true,
       validator: responsivePropValidator(Object.keys(spacing)),
-    },
+    } as PropOptions<Spacing>,
```

Comment:
I believe you missed that this prop can be an array as well
```suggestion
    } as PropOptions<Spacing | Spacing[]>,
```

Reply:
You're right! Thanks 👍🏽